src/block/accordion/edit.js

```diff
@@ -147,6 +148,17 @@ const Edit = props => {
 					<Alignment.InspectorControls />
 					<BlockDiv.InspectorControls />
 					<Advanced.InspectorControls />
+					<InspectorAdvancedControls>
```

Comment:
Move this up so that the **FAQ Schema** panel would be the first entry in the Advanced Tab if possible.